docs/book/src/developer/providers/implementers-guide/building_running_and_testing.md

```diff
@@ -30,7 +30,7 @@ kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/<ver
 
 Before you can deploy the infrastructure controller, you'll need to deploy Cluster API itself.
 
-You can [use a precompiled manifest][install], or clone [`cluster-api`][capi] and apply its manifests using `kustomize`.
+You can use a precompilaed manifest from the release page, or clone [`cluster-api`][capi] and apply its manifests using `kustomize`.
```

Comment:
```suggestion
You can use a precompiled manifest from the release page, or clone [`cluster-api`][capi] and apply its manifests using `kustomize`.
```

Reply:
Should we create an issue to replace this with clusterctl rather than manually deploying manifests (at least for the Cluster API portions)?

Reply:
It definitely needs to be revisited